lib/daru/dataframe.rb

```diff
@@ -1578,6 +1578,10 @@ def pivot_table opts={}
       raise ArgumentError,
         "Specify grouping index" if !opts[:index] or opts[:index].empty?
 
+      arr = []
+      self.size.times {|i| arr[i] = i}
+      new_dataframe_index = Daru::Index.new(arr)
+      self.index = new_dataframe_index
```

Comment:
This seems to be kind of unexpected behaviour, but I don't think this would change any of the results. This will infact solve the bug.